client/src/components/AdminPanel/AdminPanelHome/AdminPanelHome.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import './AdminPanelHome.scss';
+import { Card } from 'react-bootstrap';
+
+export default class AdminPanelHome extends React.Component {
+
+    render() {
+        const { role } = this.props;
+        return (
+            <div className="home">
+                <Card>
+                    <Card.Header>
+                        <Card.Title>
+                            Welcome to the {role} Panel
+                        </Card.Title>
+                    </Card.Header>
+                    <Card.Body>
+                        You can perform advanced actions such as bulk creating monuments or memorials from this page. Please see the sidebar to the left for advanced actions.
```

Comment:
Does this read more clearly if it's `You can perform advanced actions such as bulk creating monuments and memorials from this page...`. (Swap "or" for "and").